docs/_getting-started-macos-android.md

```diff
@@ -173,6 +173,7 @@ If everything is set up correctly, you should see your new app running in your A
 
 `npx react-native run-android` is one way to run your app - you can also run it directly from within Android Studio.
 
+> If you are getting `Error: spawnSync ./gradlew EACCES`. run `sudo chmod 755 android/gradlew` to makes gradlew files into executable. Which help code to give a run. 
```

Comment:
This should be moved to the `troubleshooting.md` page

Reply:
alright. I will post this on the troubleshoot.md